frontend/src/app/accountCreation/AccountCreationApp.tsx

```diff
@@ -36,15 +36,18 @@ const AccountCreation404Wrapper: FunctionComponent<WrapperProps> = ({
 
 const AccountCreationApp = () => {
   const dispatch = useDispatch();
-  const activationToken = useSelector((state: any) => state.activationToken);
+  const activationToken = useSelector<RootState, string>(
+    (state) => state.activationToken
+  );
 
   useEffect(() => {
     dispatch(
       setInitialState({
         activationToken: getActivationTokenFromUrl(),
       })
     );
-  });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
My understanding is it's pretty much never a good idea to disable this warning. Should be fine to just add `dispatch` to the dependency array I think

Reply:
Oh okay great! Fixed in ca28901